src/core/time.d

```diff
@@ -611,7 +622,7 @@ public:
                   subtract this $(D Duration) from.
       +/
     Duration opBinaryRight(string op, D)(D lhs) const nothrow @nogc
-        if((op == "+" || op == "-") &&
+        if((op == "+" || op == "-" || op == "%") &&
             is(_Unqual!D == TickDuration))
     {
         return Duration(mixin("lhs.hnsecs " ~ op ~ " _hnsecs"));
```

Comment:
Oops, you're right. I decided it wasn't worth supporting `TickDuration` at the point when I got to writing tests :) Should I take it out?